crates/hotshot-signature-key/src/bn254/bn254_priv.rs

```diff
@@ -54,6 +54,7 @@ impl BLSPrivKey {
     }
 }
 
+// #[allow(clippy::incorrect_partial_ord_impl_on_ord_type)]
```

Comment:
Do we want to get rid of this dead code?

Reply:
I kept this in because my local clippy needed it.  I expect once CI's clippy is upgraded we'll need to uncomment these.  :)